src/commands/data/export/tree.ts

```diff
@@ -11,10 +11,12 @@ import { orgFlags, prefixValidation } from '../../../flags.js';
 import { ExportConfig, runExport } from '../../../export.js';
 import type { DataPlanPart, SObjectTreeFileContents } from '../../../types.js';
 
+export type ExportReturnType = DataPlanPart[] | SObjectTreeFileContents;
```

Comment:
Unfortunately we are not consistent with our naming patterns for command return results.  This one would be quite different so let's go with the most common pattern; "*Result".  E.g., `ExportTreeResult`